### PR TITLE
[Intel MKL] scope_alloctor bug fix with One binary support for v2.5

### PR DESCRIPTION
**Pull request by nammbash**
Patch for scope_allocator Fix that worked in TF2.3 and TF2.4.1. 
Modifying the same patch or TF2.5 / Master with some attention to one binary support here.
Context: https://github.com/tensorflow/tensorflow/issues/34213

### CODE REVIEW

**Comment by nammbash** [comment]
@penpornk Scope_allocator bug fix for v2.5

**Comment by penpornk** [comment]
@nammbash Thank you for the PR!

Context for @ezhulenev : This is a TF-oneDNN fix for issue https://github.com/tensorflow/tensorflow/issues/34213#issuecomment-783857373

**Review by ezhulenev — APPROVED** [review]

File: tensorflow/core/grappler/optimizers/BUILD [hunk 1]
```diff
@@ -1,5 +1,5 @@
 load("//tensorflow/core/platform:rules_cc.bzl", "cc_library")
-load("//tensorflow:tensorflow.bzl", "tf_cc_test", "tf_cc_test_mkl", "tf_copts", "tf_cuda_cc_test")
+load("//tensorflow:tensorflow.bzl", "tf_cc_test", "tf_cc_test_mkl", "tf_copts", "tf_cuda_cc_test","tf_kernel_library")
```
Comment (penpornk) [inline tensorflow/core/grappler/optimizers/BUILD:2]:
```suggestion
load("//tensorflow:tensorflow.bzl", "tf_cc_test", "tf_cc_test_mkl", "tf_copts", "tf_cuda_cc_test", "tf_kernel_library")
```

**Review by penpornk — CHANGES_REQUESTED** [review]
This PR failed the [Ubuntu Sanity](https://source.cloud.google.com/results/invocations/fd97539c-57e0-48e6-81ec-dbd7d1330dc5/log) test.
```
FAIL: buildifier found errors and/or warnings in above BUILD files.
buildifier suggested the following changes:
tensorflow/core/grappler/optimizers/BUILD:
2c2
< load("//tensorflow:tensorflow.bzl", "tf_cc_test", "tf_cc_test_mkl", "tf_copts", "tf_cuda_cc_test","tf_kernel_library")
---
> load("//tensorflow:tensorflow.bzl", "tf_cc_test", "tf_cc_test_mkl", "tf_copts", "tf_cuda_cc_test", "tf_kernel_library")
```
Could you please fix the formatting?

**Comment by nammbash** [comment]
@penpornk Accepted your suggestion. Thank you. 
 We run clang internally on CPP files! Can we run buildifier BUILD file checks?

**Review by penpornk — APPROVED** [review]
Thank you for the quick fix! 
Yes, in the future, it'd be great if you could also run buildifier. It's part of the `Ubuntu Sanity` check. :)

**Pull request merged**